cloud/src/autogluon/cloud/scripts/train.py

```diff
@@ -104,9 +109,19 @@ def get_env_if_present(name):
         tune_file = get_input_path(args.tune_dir)
         tuning_data = TabularDataset(tune_file)
 
-    if args.images_dir:
-        image_compressed_file = get_input_path(args.images_dir)
-        shutil.unpack_archive(image_compressed_file)
+    if args.train_images:
+        train_image_compressed_file = get_input_path(args.train_images)
+        train_images_dir = "train_images"
+        shutil.unpack_archive(train_image_compressed_file, train_images_dir)
+        image_column = config["image_column"]
+        training_data[image_column] = [os.path.join(train_images_dir, path) for path in training_data[image_column]]
+
+    if args.tune_images:
+        tune_image_compressed_file = get_input_path(args.tune_images)
+        tune_images_dir = "tune_images"
+        shutil.unpack_archive(tune_image_compressed_file, tune_images_dir)
+        image_column = config["image_column"]
+        tuning_data[image_column] = [os.path.join(tune_images_dir, path) for path in tune_images_dir[image_column]]
```

Comment:
Similar comment to above

Reply:
`train_images_dir` should be `tune_images_dir` ;). Easy to make mistakes when copy paste haha